api.py

```
@@ -35,26 +38,76 @@ class TTSRequest(BaseModel):
+            "totalDuration": total_duration,
+            "segments": [s.dict() for s in segments],
+            "metadata": {
+                "sampleRate": tts.synthesizer.output_sample_rate,
+                "language": request.language,
+                "speaker": request.speaker if not request.speaker_wav_path else "custom",
+                "numWords": len(words),
+                "avgWordDuration": avg_word_duration
+            }
+        }
     except Exception as e:
         raise HTTPException(status_code=500, detail=str(e))
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)